### ISSUE TITLE
TranslateHandle does not fire sigRegionChangeStarted

### ISSUE THREAD

**Issue by edbarnard** [pyqtgraph/pyqtgraph#463]
When I add a translation handle to an ROI, it does not send the `sigRegionChangeStarted` that I expected given the documentation for that signal. When I drag the handle, I get `sigRegionChanged` and `sigRegionChangeFinished`, but not `sigRegionChangeStarted`

Other types of handles seem to trigger `sigRegionChangeStarted` on start of a user dragging the handle.

Tested with Python 3.5, pyqtgraph 0.10.0, PyQt5

Thanks!
-Ed

```python
import pyqtgraph as pg
from pyqtgraph.Qt import QtCore, QtGui

app = QtGui.QApplication([])

plot = pg.PlotWindow()
roi = pg.RectROI([20, 20], [20, 20], pen=(0,9))
plot.addItem(roi)

roi.addTranslateHandle((0,0))

roi.sigRegionChangeFinished.connect(lambda x: print("sigRegionChangeFinished"))
roi.sigRegionChangeStarted.connect(lambda x: print("sigRegionChangeStarted"))
roi.sigRegionChanged.connect(lambda x: print("sigRegionChanged"))

if __name__ == '__main__':
    import sys
    if (sys.flags.interactive != 1) or not hasattr(QtCore, 'PYQT_VERSION'):
        QtGui.QApplication.instance().exec_()
```

**Comment by j9ac9k** [comment]
Now that we've merged #951 we should see if we can replicate this.

**Comment by ixjlyons** [comment]
Yep looks like this was also fixed by #951
